p8e-shared/src/main/kotlin/io/provenance/p8e/shared/service/AffiliateService.kt

```diff
@@ -296,6 +296,11 @@ class AffiliateService(
                         require (response.isAcknowledged) { "ES index creation of $it was not successful" }
                     }
                 }
+
+                if (jwt != null && identityUuid != null) {
+                    keystoneService.registerKey(jwt, signingPublicKey.publicKey, ECUtils.LEGACY_DIME_CURVE, KeystoneKeyUsage.CONTRACT)
```

Comment:
do we still need keystone for anything now? this might be moving to smartkey soon too.